app/services/metrics_service.rb

```diff
@@ -166,26 +169,37 @@ def top_validation_screens(time_period)
           .pluck(Arel.sql("page, COUNT(DISTINCT assessment_code)"))
   end
 
-  def average_completion_time(range = nil)
-    return unless relevant_events(range).where(page: "view_results").any?
+  def mode_completion_time(level_of_help, range = nil)
+    completed_checks = level_of_help == :controlled ? controlled_checks_completed(range) : certificated_checks_completed(range = nil)
```

Comment:
should the range be set to nil in the certificated check method?

Reply:
Nope! silly copy-paste error